packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -259,7 +260,9 @@ export const signTransaction = (
     const decision = await dispatch(modalActions.openDeferredModal({ type: 'review-transaction' }));
     if (decision) {
         // push tx to the network
-        return dispatch(pushTransaction());
+        const result = await dispatch(pushTransaction());
+        dispatch(routerActions.goto('wallet-index', undefined, true));
```

Comment:
this action is shared between send/rbf/invity forms so redirection should be done from send component/hook instead.
another important fact is that there should be a result check before redirection, there could be an error (failed to send...)

move it to `useSendForm` hook [sign method](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/hooks/wallet/useSendForm.ts#L224) pls